README.rst

```diff
@@ -45,6 +45,50 @@ Hello World
    >>> print(f"Observed: {CLs_obs}, Expected: {CLs_exp}")
    Observed: 0.05251497423736956, Expected: 0.06445320535890459
 
+
```

Comment:
Can there be some additional text explaining what is happening so that it isn't just two code blocks one after another?

Reply:
done